framework/src/st_things/things_stack/things_stack.c

```diff
@@ -373,7 +374,7 @@ int things_start_stack(void)
 				return 0;
 			}
 			//call wifi scan ap.
-			if (!things_start_scanning_ap())
+			if (!things_start_wifi_scan())
```

Comment:
This conflicts with #2550 
please check it.

Reply:
Resolved conflict.